docs/dev_guide/ceph_s3_tests/ceph_s3_tests_guide.md

```diff
@@ -88,8 +89,26 @@ kubectl apply -f src/test/system_tests/ceph_s3_tests/test_ceph_s3_job.yml
 kubectl logs job/noobaa-tests-s3 -f
 ```
 
+#### Skipped tests
+
 We run all the tests except the tests that appear in the lists `src/test/system_tests/ceph_s3_tests/s3-tests-lists` if you would like to add or remove a test you can edit those files (and then repeat the steps starting from 'Build Core And Tester Images (Noobaa-Core)' above).
 
+Some tests are marked to be skipped in the tests code. usually because the enviorment doesn't support certain criteria. those tests will appear with a `[WARN]` tag and will be marked as "Test Skipped", for example:
+`[WARN] CONSOLE:: Test skipped: s3tests_boto3/functional/test_s3.py::test_lifecycle_transition`
+
+In the test code the function:
+`pytest.skip("")` will mark them to be skipped.
+
+#### Boto3 tests
+As mentioned in s3-tests repo (https://github.com/ceph/s3-tests#readme):
+Most of the tests have both Boto3 and Boto2 versions. Tests written in Boto2 are in the s3tests directory. Tests written in Boto3 are located in the s3test_boto3 directory.
+
+In order to only run the boto3 version, tests in the boto directory run with `-- -m 'not fails_on_rgw`:
+
+```bash
+S3TEST_CONF=your.conf tox -- -m 'not fails_on_rgw' s3tests_boto3/functional::test_name
+```
+
```

Comment:
asked about fails_on_rgw in this PR:
https://github.com/ceph/s3-tests/pull/482/files#diff-7b3ed02bc73dc06b7db906cf97aa91dec2b2eb21f2d92bc5caa761df5bbc168fR44-R48

will water for ceph team for response. if they will not respond I will remove this section for now. I will also remove the note in --disable-pytest-warnings as it will not be needed anymore

Reply:
after talking to one of the ceph people, it appears that this flag is for filtering out tests that don't pass on ceph rgw, and have nothing to do with boto3 tests. as the filtered tests pass on our system. I have removed this filter from test_ceph_s3 and removed the section about it from the documentation, also removed the note in --disable-pytest-warnings as it is not needed anymore